spec/javascript/components/ShopShowSpec.js

```diff
@@ -0,0 +1,85 @@
+import ShopShow from '../../../app/javascript/react/containers/ShopShow'
+import DonutTile from '../../../app/javascript/react/components/DonutTile'
+import fetchMock from 'fetch-mock'
+
+describe('ShopShow', () => {
+  let data,
+      wrapper;
+
+  beforeEach(() => {
+  data = {
+    shop: {
+      id: 13,
+      name: "Honeydew Donuts",
+      address: "1600 Pennsylvia Ave NW",
+      city: "Washington, DC",
+      state: "DC",
+      url: "www.honeydewrules.org",
+      phone: "111-111-1112",
+      donuts: [
+        {
+          id: 9,
+          name: "Jelly Donut",
+          reviews: [
+            {
+              id: 6,
+              rating: 2,
+              body: "this is a very mediocre and not exciting donut",
+              price_range: "2",
+              photo_url: "https://www.theflavorbender.com/wp-content/uploads/2014/09/Simpsons-Doughnuts-4238-Copy-1.jpg",
+              donut_id: 9,
+              created_at: "2018-10-23T20:05:56.382Z",
+              updated_at: "2018-10-23T20:05:56.382Z",
+              shop_id: 12
+            }
+          ],
+          shop: {
+            id: 13,
+            name: "Honeydew Donuts",
+            address: "1600 Pennsylvia Ave NW",
+            city: "Washington, DC",
+            state: "DC",
+            url: "www.honeydewrules.org",
+            phone: "111-111-1112",
+            created_at: "2018-10-23T20:05:14.954Z",
+            updated_at: "2018-10-23T20:05:14.954Z"
+          }
+        }
+      ],
+      reviews: [
+        {
+          rating: 2,
+          body: "this is a very mediocre and not exciting donut",
+          price_range: "2",
+          photo_url: "https://www.theflavorbender.com/wp-content/uploads/2014/09/Simpsons-Doughnuts-4238-Copy-1.jpg"
+        }
+      ]
+    }
+  }
+  fetchMock.get('/api/v1/shops/13', {
+    status: 200,
+    body: data
+  })
+    wrapper = mount(
+      <ShopShow
+        params = {{ id: 13}}
+      />
+    );
+  });
+
+  afterEach(fetchMock.restore)
+
+    it('should render a donut tile component', (done) => {
+      setTimeout(() => {
+          expect(wrapper.find(DonutTile)).toBePresent();
+          done()
+      }, 0)
+    });
+
+    it('should render an h1', (done) => {
+      setTimeout(() => {
+        expect(wrapper.find('h1')).toBePresent()
+        done()
+      }, 0)
+    })
+  });
```

Comment:
Here, you should be checking that this component has received some of the info from beforeEach is rendered.
Either test that specific pieces of shop information have been rendered on the DOM, or that props have been passed into the component correctly. 

Please go back to the enzyme readings and add a little bit more :/

Reply:
addressed in controller_enzyme_updates branch <3